core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2031,6 +2031,13 @@ package object config {
       .intConf
       .createWithDefault(1)
 
+  private[spark] val IO_COMPRESSION_LZF_PARALLEL =
```

Comment:
Should we add an explanation of this configuration in `configuration.md`

Reply:
thanks, addressed